.github/dependabot-bot.yml

```diff
@@ -11,4 +11,4 @@ safe:
   - "render-engine-markdown"
   - "rich"
   - "typer"
-  - "watchdog"
\ No newline at end of file
+  - "watchdog"
```

Comment:
ultimately there is a goal to remove dependabot and instead update our environment as a part of pre-commit